src/apps/main/switcher.cpp

```diff
@@ -168,7 +168,7 @@ void OswAppSwitcher::paginationEnable() {
 void OswAppSwitcher::sleep(OswHal* hal, boolean fromButton) {
   hal->gfx()->fill(rgb565(0, 0, 0));
   hal->flushCanvas();
-
+  
```

Comment:
Could you clean that? Maybe even with an `git commit --amend`? Thx.

Reply:
i think i was able to amend? cant really tell .. pls ack when able. thank you